cl/simple_pages/templates/advanced_search.html

```diff
@@ -84,6 +84,15 @@ <h2>Date Queries</h2>
       </code>
     </p>
     <p>We are working to make this easier for our users.</p>
+
+    <h2>Related opinions <code>related:&lt;id&gt;</code></h2>
+    <p>
+      Find opinions whose content is most related to the content of the given opinion.
+      Replace <code>&lt;id&gt;</code> with the ID of the opinion for which you want to retrieve related opinions.
+    </p>
+    <p>
+      The opinion ID can be found in the URL of the corresponding opinion: <code>courtlistener.com/opinion/&lt;id&gt;/...</code>
```

Comment:
Is this right? The ID in the URL is the `cluster` ID, not the opinion ID, and there can be multiple `opinion`s in a given cluster on a given page. This might be a larger issue, but I think you're using opinions for MLT when clusters might make more sense from a quality perspective. From a simplicity perspective, the opinion makes sense, but we don't expose the opinion ID anywhere, and I'm also not sure how we'd handle the fact that multiple opinions can go onto a given page.

Reply:
Isn't it? The opinion detail page (`opinion_page/views:view_opinion`) queries the `OpinionCluster` model?

Reply:
Let's revisit one more time.

Reply:
What's the issue?

Reply:
I'll remove the URL-based explanation and replace it with a references + screenshot of the UI.

Reply:
Looks like this still needs resolution too.

Reply:
Added UI explanation.